crates/basilica-sdk-python/examples/start_secure_cloud_gpu_rental.py

```
@@ -0,0 +1,305 @@
+
+    # Step 4: Wait for rental to be ready
+    print("\n[Step 4] Waiting for rental to be ready...")
+    try:
+        ready_rental = wait_for_rental_ready(client, rental.rental_id)
+    except (TimeoutError, RuntimeError) as e:
+        print(f"\nError: {e}")
+        print("Attempting to stop the rental...")
+        try:
+            client.stop_secure_cloud_rental(rental.rental_id)
+            print("Rental stopped.")
+        except Exception as stop_error:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)